docs/.vitepress/data/glossary.data.ts

```diff
@@ -0,0 +1,23 @@
+export default {
+  load() {
+    return {
+      "buffer": "Data of a loaded file and additional metadata, associated with one or more views. The distinction between buffer and :view: is technical. Most of the time, both terms can be used interchangeably.",
```

Comment:
I preferred the YAML frontmatter over this JS dict, but i assume we need this as a data file to be able to check for a term's existence?

Reply:
Yaml frontmatter isn’t accessible from another page with vitepress like it was with Vuepress. Not in a documented way.

Reply:
Yes, because vitepress does not have a documented way for other pages to access another pages frontmatter